tests/assets/default.conf

```diff
@@ -25,3 +25,7 @@ appendonly no
 appendfsync everysec
 no-appendfsync-on-rewrite no
 activerehashing yes
+
```

Comment:
> https://github.com/redis/redis/runs/4576917235?check_suite_focus=true#step:8:149

look like we also need to enable `protected` and `debug` in sentinel test
> debug: Master reboot in very short time: ERR DEBUG command not allowed. If the enable-debug-command option is set to "local",you can run it from a local connection, otherwise you need to set this option in the configuration file, and then restart the server.

> protected: SDOWN is triggered by misconfigured instance replying with errors: ERR CONFIG SET failed (possibly related to argument 'dir') - can't set protected config

work fine in local: https://github.com/redis/redis/commit/4c87042760d0972fb9ea6f35f1d815611fabbbb4
i will try to trigger a Daily in my fork

Reply:
@enjoy-binbin can you make a PR to solve these?